vm/systemSmartContracts/staking.go

```diff
@@ -84,11 +84,23 @@ func (r *stakingSC) Execute(args *vmcommon.ContractCallInput) vmcommon.ReturnCod
 		return r.isStaked(args)
 	case "setStakeValue":
 		return r.setStakeValueForCurrentEpoch(args)
+	case "jail":
+		return r.jail(args)
+	case "unJail":
+		return r.unJail(args)
 	}
 
 	return vmcommon.UserError
 }
 
+func (r *stakingSC) unJail(args *vmcommon.ContractCallInput) vmcommon.ReturnCode {
+	return vmcommon.Ok
```

Comment:
Same as above, valid also for the next function

Reply:
implemented